src/github/folderRepositoryManager.ts

```diff
@@ -868,7 +868,7 @@ export class FolderRepositoryManager implements vscode.Disposable {
 			return [];
 		}
 
-		const localBranches = this.repository.state.refs
+		const localBranches = (await this.repository.getRefs({}))
```

Comment:
Given that we seem to be doing the filtering, I would suggest that we pass a `pattern` here so that we only get the refs that we are interested in. (ex: `getRefs({ pattern: 'refs/heads/*' }));`

Reply:
Looks like `refs/heads/` works 👍